### PR TITLE
Add More Flow Units

### PR DESCRIPTION
**Pull request by gojanpaolo**
Fixes https://github.com/angularsen/UnitsNet/issues/360

### CODE REVIEW

File: UnitsNet/UnitDefinitions/Flow.json [hunk 1]
```diff
@@ -101,6 +173,18 @@
...
+    },
+	{
+      "SingularName": "UsGallonsPerSecond",
+      "PluralName": "UsGallonsPerSecond",
+      "FromUnitToBaseFunc": "x/264.1720523581484",
+      "FromBaseToUnitFunc": "x*264.1720523581484",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "galUS/s" ]
```
Comment (eriove) [inline UnitsNet/UnitDefinitions/Flow.json]: In volume we've written `gal (U.S.)`. Then `gal (U.S.)/s` would be consistent here, but that seems awkward. @angularsen what do you think?

File: UnitsNet/UnitDefinitions/Flow.json [hunk 2]
```diff
@@ -113,6 +197,18 @@
...
+    },
+	{
+      "SingularName": "UsGallonsPerHour",
+      "PluralName": "UsGallonsPerHour",
+      "FromUnitToBaseFunc": "x/951019.38848933424",
+      "FromBaseToUnitFunc": "x*951019.38848933424",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "galUS/h" ]
```
Comment (eriove) [inline UnitsNet/UnitDefinitions/Flow.json]: Same comment regarding `gal (U.S.)` here

File: UnitsNet/UnitDefinitions/Flow.json [hunk 3]
```diff
@@ -47,6 +59,66 @@
...
+    },
+    {
+      "SingularName": "CubicFootPerHour",
+      "PluralName": "CubicFeetPerHour",
+      "FromUnitToBaseFunc": "x*7.8657907199999087346816086183876e-6",
+      "FromBaseToUnitFunc": "x/7.8657907199999087346816086183876e-6",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "cf/hr" ]
```
Comment (eriove) [inline UnitsNet/UnitDefinitions/Flow.json]: The official abbreviation of cubic foot is ft³ and cf isn't listed on Wikipedia. I would suggest to change this to ft³/h and perhaps add this as an alternative

Comment (gojanpaolo) [reply UnitsNet/UnitDefinitions/Flow.json]: Will do. :)

File: UnitsNet/UnitDefinitions/Flow.json (hunk 1 above)
Comment (gojanpaolo) [reply UnitsNet/UnitDefinitions/Flow.json]: We actually display US Gallons as gal (US).
Takes less space and seems to be clear enough.
Also less awkward when used in Flow gal (US)/s

We also display US Survey Feet as ft (US) ... ft (US)/s

File: UnitsNet/UnitDefinitions/Flow.json [hunk 4]
```diff
@@ -47,6 +59,66 @@
...
+    },
+    {
+      "SingularName": "CubicFootPerHour",
+      "PluralName": "CubicFeetPerHour",
+      "FromUnitToBaseFunc": "x*7.8657907199999087346816086183876e-6",
+      "FromBaseToUnitFunc": "x/7.8657907199999087346816086183876e-6",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "ft³/h", "cf/hr" ]
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/Flow.json]: Is `cf/hr` used? Having it here means you expect to parse it.
If we keep this, should the same be added for cubic yards per hour below?

File: UnitsNet/UnitDefinitions/Flow.json (hunk 1 above)
Comment (angularsen) [reply UnitsNet/UnitDefinitions/Flow.json]: It is awkward, but at least it is consistent. I think it's fine.

File: UnitsNet/UnitDefinitions/Flow.json (hunk 4 above)
Comment (gojanpaolo) [reply UnitsNet/UnitDefinitions/Flow.json]: `cf/hr` was already there before this PR so I didn't remove it. I suggest we put that task of removing it in the wishlist for breaking changes.

Comment (angularsen) [reply UnitsNet/UnitDefinitions/Flow.json]: Ah I missed that. Good point, if you could add it to the wishlist that would be good!

Comment (gojanpaolo) [reply UnitsNet/UnitDefinitions/Flow.json]: Done! :)

**Comment by angularsen** [comment]
Alrighty, this should be good to go then

**Comment by angularsen** [comment]
Nuget 3.87 on the way out

**Comment by gojanpaolo** [comment]
Thank you! :100:

**Pull request merged**